<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>350</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Order Scaner</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="BG_Char">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="title">
         <string>Char</string>
        </property>
        <widget class="QComboBox" name="QCombo_Chars">
         <property name="geometry">
          <rect>
           <x>25</x>
           <y>18</y>
           <width>150</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="GB_Load">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>190</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>50</height>
         </size>
        </property>
        <property name="title">
         <string>Load orders from</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <widget class="QPushButton" name="BTN_LoadAPI">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>15</y>
           <width>80</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>API</string>
         </property>
        </widget>
        <widget class="QPushButton" name="BTN_LoadFile">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>15</y>
           <width>80</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>File</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="GB_Scaner">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>190</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="title">
         <string>Scaner</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <widget class="QPushButton" name="BTN_ScanStart">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>15</y>
           <width>80</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
        <widget class="QPushButton" name="BTN_ScanStop">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>15</y>
           <width>80</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="Label_TableOrders_Sell">
        <property name="text">
         <string>Sell orders</string>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <property name="indent">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="Table_OrdersSell">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>18</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>18</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Label_Table_OrdersBuy">
        <property name="text">
         <string>Buy orders</string>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <property name="indent">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="Table_OrdersBuy">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>18</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>18</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuApi_Managment">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="action_APIManager"/>
    <addaction name="action_CharManager"/>
    <addaction name="separator"/>
    <addaction name="action_Settings"/>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menuApi_Managment"/>
  </widget>
  <action name="actionAPI_Manager">
   <property name="text">
    <string>API Manager</string>
   </property>
  </action>
  <action name="action_APIManager">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>API Manager</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_Settings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="action_CharManager">
   <property name="text">
    <string>Character Manager</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
